server/src/linker.js

```diff
@@ -29,7 +29,7 @@ exports.staticLink.simple = function(resource) {
 };
 
 exports.staticLinkWithHost = function(req, resource) {
-  return req.protocol + "://" + req.headers.host + exports.staticLink(resource);
+  return req.protocol + "://" + req.headers.host + `${resource}?rev=${gitRevision}`;
```

Comment:
Shouldn't it use `req.cdn` if it's available? With this patch `staticLinkWithHost` and `staticLink` aren't that much different anymore (except of course when the CDN isn't configured)